CHANGES.md

```diff
@@ -5,8 +5,9 @@ unreleased
   + merlin binary
     - Improve error messages for missing configuration reader (#1669)
     - Fix regression causing crash when using ppxes under Windows (#1673)
-
-
+  + editor modes
+    - emacs: fix/improve keybindings (#1668, fixes #1386).
+      See <https://github.com/ocaml/merlin/issues/1386#issuecomment-1701567716>
```

Comment:
One last thing: we should probably list the binding changes here in addition to the link.